### PR TITLE
8 write test cases

### LINKED ISSUE #8: Write test cases
The test  _**must**_ include information about the test environment. (i.e. Versions, Programs, OS, etc.)

### PR DESCRIPTION
**Pull request by h0peRL**

### CODE REVIEW

**Review by mnaray — CHANGES_REQUESTED** [review]
Overall your PR seems fine to me, however there are some changes I require you to make before merging it into main. Please make sure to enforce these points carefully before requesting another review from me or @EllisArn. 

I corrected some of the typos I found on the fly, although I need you to fix the bigger faults listed below, as I am working on another issue.

Test cases to correct:
- [x] 1.1 => The test environment is Visual Studio 2022 plus its version number. .NET 8.0 is the runtime
- [x] 4.1 => Please define an API endpoint to use (like in 3.1)
- [x] 5.1 => Please define the script more specifically in the requirement column.
- [x] 6.1 => Please define the script more specifically in the requirement column.
- [ ] 9.1 => Could that be a nonexistent route? Please Check the Controller's class diagram. There was no intention to create a help command.
- [x] 10.1 => Please provide a concrete endpoint to run this test on.
- [x] 13.1 => A glossary contains translations **and** explanations.

Further problems related to this PR:
- [x] Missing definition of a test environment. Please include all the version/build numbers of the programs and runtimes listed in it. See old projects for examples.
- [x] Requirement 11 is wrong. This would be an opportunity to fix it. We won't necessarily be using commands on our REST API. Instead, we'll offer endpoints through controller objects. To document those, we could add a .../endpoint/info route to (or similar) to every endpoint.
- [x] Column names of the test case table are written in German. Please translate those to English.

_Additional note:_
Please make sure to pay attention to accidental removal of indentations by the markdown editor. I had to manually readjust the indentations of the blue boxes. This requires you to take a peek at your diff before committing and pushing your changes.

**Review by h0peRL — COMMENTED** [review]
I applied all the requested changes.

File: docs/project/documentation.md [hunk 1]
```diff
@@ -6,32 +6,29 @@
...
-| 11           | Optional               | Quality                 | Every command should have an explanation of the syntax                                         |
-| 12           | Mandatory              | Functional              | Every endpoint should require a JWT-Bearer-Token (except login and register)                   |
-| 13           | Mandatory              | Marginal                | There must be a UML class diagram.                                                             |
-| 14           | Mandatory              | Marginal                | The program and documentation should be in English                                             |
-| 15           | Mandatory              | Marginal                | There should be a glossary with German translations                                            |
+| **Req. No.** | **Mandatory/Optional** | **Func./Qual./Margin.** | **Requirement**                                                                                |     |
+| :----------- | ---------------------- | :---------------------- | ---------------------------------------------------------------------------------------------- | --- |
```
Comment (mnaray) [inline docs/project/documentation.md]: I think you might have added an accidental column here.

File: docs/project/documentation.md [hunk 2]
```diff
@@ -63,20 +60,64 @@
 
 ???+ info "Total Time Spent (estimate)"
 
-    Here is the calculation of how many lessons the project should take to implement. Please note, that a single lesson equals 45 minutes
```
Comment (mnaray) [inline docs/project/documentation.md]: You removed the needed indentation again in this section.

File: docs/project/documentation.md [hunk 3]
```diff
@@ -63,20 +60,64 @@
...
+```
+team members: 3
+total weeks: 7
+lessons per week: 5
+lessons per member: 7 * 5 = 35
+total lessons: 35 * 3 = 105
+```
 
-    **The total amount of lessons should be 105** as every one of the 3 members works 5 lessons per week over the course of 7 weeks.
+**The total amount of lessons should be 105** as every one of the 3 members works 5 lessons per week over the course of 7 weeks.
```
Comment (mnaray) [inline docs/project/documentation.md]: Everything up to this line should be included.

**Review by mnaray — CHANGES_REQUESTED** [review]
- [x] Remove the empty column from the requirements.
- [x] Fix the indentation issue with the blue box for the lesson calculations. (Please remember to check your diff before committing)
- [x] Please make sure that you use version numbers that are up-to-date.
- [x] Fix the Bulletpoint's indentation.
- [x] 9.1's route would be .../booking/book

**Review by mnaray — APPROVED** [review]

**Pull request merged**